merge_duplicate_contacts/wizard/merge_contact.py

```diff
@@ -0,0 +1,935 @@
+import itertools
+import logging
+import operator
+from ast import literal_eval
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError, ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
```

Comment:
Could you put one model per file ?

The res.partner code should be moved in models